main.js

```diff
@@ -1,5 +1,49 @@
-const text = "hello world";
+//Переменные
+let user = "John Doe";
+console.log(user);
 
-console.log(text);
+const student = "Michael";
+console.log(student);
 
-alert(text);
+user = student; /* Michael */
+console.log(student);
+
+//Примитивы
+
+let test = 1;
+test++;
+test = test + "1"; /*21, because "1" is a string*/
+console.log(test);
+test = test - 1; /*20*/
+console.log(test);
+test = Boolean("test");
+/* Eslint "comma-dangle": ["error", "always-multiline"] dont work, fix later*/
+console.log(test);
+/*true, because not empty string, false if test = Boolean("") */
+
+//Массивы
+
+let result = 1;
+const arr = [2, 3, 5, 8];
+for (let i = 0; i < arr.length; i++) {
+  result = result * arr[i];
+}
+console.log(result);
+
+console.log("---------------");
+
+const arr1 = [2, 5, 8, 15, 0, 6, 20, 3];
+for (let i = 0; i < arr1.length; i++) {
+  if (5 < arr1[i] && arr1[i] < 10) {
+    console.log(arr1[i]);
+  }
+}
+
+console.log("---------------");
+
+const arr2 = [2, 5, 8, 15, 0, 6, 20, 3];
+for (let i = 0; i < arr2.length; i++) {
```

Comment:
It's working incirrectly, you compare position of item in array, but not item in arr

Reply:
Corrected